### ISSUE TITLE
Equalized containers aren't re-equalized when toggling state of expandable/collapsible children

### ISSUE THREAD

**Issue by EricDunsworth** [wet-boew/wet-boew#6418]
Not sure how common of a scenario this is, but I figured I'd open an issue about it anyway.

**Test procedures:**
1. Dump the sample code into a WET 4.0 page template. It contains two equalized containers.
2. Open it up in a browser. Both containers will initially be equalized.
3. Expand the second container's ``<details>`` element. The first container won't resize itself to be consistent with the second container's revised height.

**Sample code:**
```
<div class="wb-eqht row">
	<section class="well col-sm-6">
		<h2>Container 1</h2>
		<p>...</p>
	</section>
	<section class="well col-sm-6">
		<h2>Container 2</h2>
		<details>
			<summary>Expandable/collapsible content</summary>
			<p>...</p>
			<p>...</p>
			<p>...</p>
		</details>
	</section>
</div>
```

**Screenshots:**
![equalize_even](https://cloud.githubusercontent.com/assets/1907279/5620278/61755114-94f8-11e4-82eb-df3e817463ad.png)

![equalize_mismatch](https://cloud.githubusercontent.com/assets/1907279/5620281/63c8fc18-94f8-11e4-96e0-35c50d5a7b9a.png)

**Comment by thomasgohard** [comment]
It wouldn't because it needs an event to trigger a re-equalisation. The only events currently supported are the page resize events.

If we had a container resize event, we could hook into that. Or if the toggling of the display/summary triggers an event.

**Comment by LaurentGoderre** [comment]
I think you probably will need to implement your own code to bridge the gap here

**Comment by pjackson28** [comment]
It wouldn't be too hard as you could trigger a resize event on every click of the summary elements in question.

**Comment by EricDunsworth** [comment]
I'll try to look into it when I have a chance. Btw should this issue be tagged with an expand/collapse label as well?

**Comment by LaurentGoderre** [comment]
Not sure this is a bug though, we should probably remove that label

**Comment by hsrudnicki** [comment]
Can this issue be closed or otherwise?

**Comment by LaurentGoderre** [comment]
Hmmmit is possible that this can be fixed without JavaScript just by changing the flexbox attributes

**Comment by EricDunsworth** [comment]
@LaurentGoderre Flexbox isn't currently used for equal heights in WET 4.0, but IMO it's be awesome to implement it and only use the existing plugin as a fallback for IE9-9.

Btw is anything in place atm for plugins to easily detect IE9?

**Comment by LaurentGoderre** [comment]
Yes WET core has properties for it and you can use the selectors on the HTML tag for CSS

**Comment by EricDunsworth** [comment]
FYI me and @ElismaD are currently looking leveraging flexbox for equalization :).

**Comment by hsrudnicki** [comment]
Based on the above discussion, this issue seems to be resolved and can be closed or otherwise? If you think strongly otherwise, please respond/update this issue's status, thank you

**Comment by EricDunsworth** [comment]
@hsrudnicki This issue is still valid. It still hasn't been fixed.

Several months ago I made a commit that started to lay the groundwork towards using flexbox CSS for equalization. That commit only exists in my fork of the wet-boew repo at this time. I was in a rush when I committed it, so its commit message isn't very detailed. My intent at the time was to overwrite that commit as I got around to making enhancements, but haven't had the time to work on it in a while.

**What my commit achieved:**

* Added flexbox CSS support for all of the equalization demo page's examples (only tested in Firefox)
* Modifications to make testing the CSS easier:
  * Temporarily disabled the equalization plugin
  * Modified the example page's CSS to make testing easier and to fix problems with the CSS solution
  * Attempted to get flexbox CSS working with the tabbed interface plugin without success (I commented-out those CSS changes)

**Things that are still needed:**

* Support for IE10/11-Edge (they support flexbox but use outdated CSS syntax for it)
* Possibly disable equalization CSS in small viewports
* Only enable the equalization JS plugin if it's actually needed (i.e. if the current browser lacks native flexbox CSS support or if the tabbed interface is used)
* Undo temporary changes I made to make testing easier
* Possibly enhance the equalization JS plugin to re-equalize under more circumstances (like when toggling expand/collapse states of ``<details>`` elements)
* Fix equalization JS support in the tabbed interface (if it's still broken)
* Maybe update documentation to explain under which circumstances flexbox CSS is used vs the JS plugin

As it stands right now, my plugin is in no state to merge into wet-boew, unless we place it in a dedicated feature branch. IMO this issue can't be considered resolved until all the items in the second list have been addressed and merged into the official wet-boew repo.

**Comment by hsrudnicki** [comment]
Based on the last comment, re-opened, thank you @EricDunsworth for the issue's update

**Comment by hsrudnicki** [comment]
Have been done the things listed in last Eric's comment? if "yes" then can be this issue closed as resolved/fixed or if otherwise? Either way, please update this issue's status again, thank you

**Comment by hsrudnicki** [comment]
Please update this issue's status whether this is still an issue or can be this closed? Thank you

**Comment by hsrudnicki** [comment]
@EricDunsworth : Is this still an issue? Either way, please update the status of this in regards to the current 2016 year, thank you

**Comment by EricDunsworth** [comment]
@hsrudnicki This issue is still valid, just like all of my other open issues. Sorry, but I don't have enough capacity to provide regular status updates on all of my issues. If a stale issue doesn't appear to be resolved based on its comments or related pull requests, that doesn't necessarily mean the issue is no longer relevant. I'd recommend against closing valid-looking issues solely due to lack of activity.

**Comment by hsrudnicki** [comment]
Up to now, none of all issues has been closed "solely due to lack of activity", always had be other clue(s)/comment(s) supporting this action. Anyway, you always can respond to re-open a "closed" issue. Thank you

**Comment by delisma** [comment]
I think this issue would be solvable with [CSS grid layout](https://www.w3.org/TR/css-grid-1/) once it moves from draft to RC and it's widely adopted by browser.
